webapp/.env.example

```diff
@@ -1,15 +1,14 @@
 # Required Variables
 # If you add any new required variables, make sure you update the variables list in the checkEnv.ts file as well.
 REACT_APP_BACKEND_URI=https://localhost:40443/
-REACT_APP_AAD_AUTHORITY=https://login.microsoftonline.com/common
-REACT_APP_AAD_CLIENT_ID=
 
-# Optional Variables
-REACT_APP_SK_API_KEY=
-
-# Authorization scopes to access webapi when using Azure AD authorization.
+# To enable authorization using Azure Active Directory, uncomment the following variables
 # See paragraph "(Optional) Enable backend authorization via Azure AD" in README.md for details and setup
-REACT_APP_AAD_API_SCOPE=
+# REACT_APP_AUTH_TYPE=AzureAd
+# REACT_APP_AAD_AUTHORITY=https://login.microsoftonline.com/common
```

Comment:
We mention "common" here...  Shouldn't we say something like [YOUR_TENANT_ID] instead?